src/core_plugins/kbn_vislib_vis_types/public/area.js

```diff
@@ -150,7 +150,7 @@ export default function PointSeriesVisType(Private) {
           name: 'group',
           title: 'Split Series',
           min: 0,
-          max: 1,
+          max: 3,
```

Comment:
Why are only 3 splits allowed? Should this be made configurable via an advanced setting?

Reply:
we are guessing it doesn't make much sense at this point ... lets wait for somebody to ask for more than 3